overlord/configstate/configcore/services.go

```diff
@@ -62,16 +61,20 @@ func switchDisableService(service, value string) error {
 }
 
 // services that can be disabled
-var services = []string{"ssh", "rsyslog"}
-
 func handleServiceDisableConfiguration(tr Conf) error {
-	for _, service := range services {
-		output, err := coreCfg(tr, fmt.Sprintf("service.%s.disable", service))
+	for _, service := range []struct {
+		cfgName     string
+		systemdName string
+	}{
+		{"ssh", "sshd.service"},
+		{"rsyslog", "rsyslog.service"},
+	} {
```

Comment:
This form of in-line for loop seems to almost always hinder readability compared to the split version, perhaps because it creates multiple indented blocks where each carry a different meaning, and the very first line is in fact most related to the last block.

Comparing with, for example:
```
        var services = []struct{ configName, systemdName string }{
                {"ssh", "sshd.service"},
                {"rsyslog", "rsyslog.service"},
        }
        for _, service := range services {
                (...)
        }